app/src/main/java/com/framgia/soundcloud/service/MusicService.java

```diff
@@ -116,6 +141,11 @@ public void onPrepared(MediaPlayer mp) {
 
     @Override
     public void onDestroy() {
+        try {
```

Comment:
WARNING: [CallSuperLast] super should be called at the end of the method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/android.html#CallSuperLast